libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -40,7 +40,7 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("CAPACITY", 7, AP_BattMonitor_Params, _pack_capacity, AP_BATT_MONITOR_BATTERY_CAPACITY),
 
-#ifndef HAL_BUILD_AP_PERIPH
+#if AP_BATTERY_WATT_MAX_ENABLED
```

Comment:
The endif on 51 needs updating to match.

Reply:
fixed